packages/decorators/src/djs-decorators.ts

```diff
@@ -21,13 +23,29 @@ const DMAvailablePermissions = new Permissions(
 	]).bitfield & Permissions.ALL
 );
 
+const DMAvailableUserPermissions = new Permissions(
+	~new Permissions([
+		//
```

Comment:
is this still needed?

Reply:
I guess not, the array is long enough so prettier isn't being a bitch about it.